server/storages/mysql.go

```diff
@@ -0,0 +1,254 @@
+package storages
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/jitsucom/jitsu/server/identifiers"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/jitsucom/jitsu/server/adapters"
+	"github.com/jitsucom/jitsu/server/events"
+	"github.com/jitsucom/jitsu/server/logging"
+	"github.com/jitsucom/jitsu/server/schema"
+)
+
+//MySQL stores files to MySQL in two modes:
+//batch: (1 file = 1 statement)
+//stream: (1 object = 1 statement)
+type MySQL struct {
+	Abstract
+
+	adapter                       *adapters.MySQL
+	processor                     *schema.Processor
+	streamingWorker               *StreamingWorker
+	usersRecognitionConfiguration *UserRecognitionConfiguration
+	uniqueIDField                 *identifiers.UniqueID
```

Comment:
You can omit: `uniqueIDField`, `staged`, and `cachingConfiguration` because they are in Abstract.

Reply:
removed properties from Abstract struct